submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/request/ExtSubmitRequest.kt

```diff
@@ -3,10 +3,10 @@ package ac.uk.ebi.biostd.persistence.common.request
 import ebi.ac.uk.extended.model.ExtSubmission
 
 data class ExtSubmitRequest(
-    val submission: ExtSubmission,
+    val key: String? = null,
```

Comment:
This is just the request object to create a request for an ExtSubmission which is why [here](https://github.com/EBIBioStudies/biostudies-backend-services/blob/feature/pivotal-%23184033688-request-draft-merge/submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submitter/LocalExtSubmissionSubmitter.kt#L66) I'm checking if it's null and creating a new id.

It was that or explicitly passing a `TMP_${Instant.now()}` wherever the submit extended methods are used. I decided for the first approach because:
1. Submissions with extended bypass the draft/validation stages so it won't have a key
2. There's a single point where the key is potentially set